aiobotocore/client.py

```diff
@@ -222,6 +235,7 @@ def get_paginator(self, operation_name):
                 self._cache['page_config'][actual_operation_name])
             return paginator
 
+    @asyncio.coroutine
     def close(self):
         """Close all http connections"""
-        self._endpoint._aio_seesion.close()
+        yield from self._endpoint._aio_session.close()
```

Comment:
btw, any reason to not do this automatically from the __del__ with a loop.run_until_complete?   Otherwise you get a warning from aiohttp.